sage-ios/SAGE/SAGE/SignUpTableViewController.swift

```diff
@@ -112,26 +107,35 @@ class SignUpTableViewController: UITableViewController, UINavigationBarDelegate
     }
 
     override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCellWithIdentifier("Cell", forIndexPath: indexPath)
+        
         if self.modalType == ContentType.School {
-            cell.textLabel?.text = self.schools[indexPath.row]
-        } else if self.modalType == ContentType.Hours {
+            let school = self.schools[indexPath.row]
+            var cell = self.tableView.dequeueReusableCellWithIdentifier("BrowseSchoolsCell")
+            if cell == nil {
+                cell = SchoolsTableViewCell(style: .Subtitle, reuseIdentifier: "BrowseSchoolsCell")
+            }
+            (cell as! SchoolsTableViewCell).configureWithSchool(school)
+            return cell!
+
+        } else {
+            let cell = tableView.dequeueReusableCellWithIdentifier("Cell", forIndexPath: indexPath)
```

Comment:
Yeah, we're using different cell classes so I wanted to keep a different identifier